doc/start/quick-cephfs.rst

```diff
@@ -3,117 +3,115 @@
 ===================
 
 To use the :term:`CephFS` Quick Start guide, you must have executed the
-procedures in the `Storage Cluster Quick Start`_ guide first. Execute this quick
-start on the Admin Host.
+procedures in the `Storage Cluster Quick Start`_ guide first. Execute this
+quick start on the admin host.
 
 Prerequisites
 =============
 
-#. Verify that you have an appropriate version of the Linux kernel. 
+#. Verify that you have an appropriate version of the Linux kernel.
    See `OS Recommendations`_ for details. ::
-   
+
 	lsb_release -a
 	uname -r
 
-#. On the admin node, use ``ceph-deploy`` to install Ceph on your 
+#. On the admin node, use ``ceph-deploy`` to install Ceph on your
    ``ceph-client`` node. ::
 
 	ceph-deploy install ceph-client
 
+#. Optionally, if you want a FUSE-mounted file system, you would need to
+   install ``ceph-fuse`` package as well.
 
 #. Ensure that the :term:`Ceph Storage Cluster` is running and in an ``active +
    clean``  state. Also, ensure that you have at least one :term:`Ceph Metadata
-   Server` running. :: 
+   Server` running. ::
 
 	ceph -s [-m {monitor-ip-address}] [-k {path/to/ceph.client.admin.keyring}]
 
 
-Create a Filesystem
-===================
+Create a File System
+====================
 
 You have already created an MDS (`Storage Cluster Quick Start`_) but it will not
-become active until you create some pools and a file system.  See :doc:`/cephfs/createfs`.
-
-::
+become active until you create some pools and a file system.  See
+:doc:`/cephfs/createfs`.::
 
-    ceph osd pool create cephfs_data <pg_num>
-    ceph osd pool create cephfs_metadata <pg_num>
+    ceph osd pool create cephfs_data 32
+    ceph osd pool create cephfs_metadata 32
     ceph fs new <fs_name> cephfs_metadata cephfs_data
 
+Quick word about Pools and PGs
+------------------------------
+Since the default replication number/size is 3, you'd need 3 OSDs to get
+``active+clean`` for all PGs. Alternatively, you may reduce the replication
+number for the pool to match the number of OSDS::
 
-Create a Secret File
-====================
-
-The Ceph Storage Cluster runs with authentication turned on by default. 
-You should have a file containing the secret key (i.e., not the keyring 
-itself). To obtain the secret key for a particular user, perform the 
-following procedure: 
-
-#. Identify a key for a user within a keyring file. For example:: 
+    ceph osd pool set data size {number-of-osds}
+    ceph osd pool set meta size {number-of-osds}
```

Comment:
Are these commands correct? In the earlier example you created pools called "cephfs_data" and "cephfs_metadata", but here you seem to be setting the size on pools called "data" and "meta". It'd be good to be consistent, or explain how these commands are related if it is correct.

Reply:
>  or explain how these commands are related if it is correct.

I've mentioned how's the command related in the paragraph right before it -

```
Since the default replication number/size is 3, you'd need 3 OSDs to get
``active+clean`` for all PGs. Alternatively, you may change the replication
number for the pool to match the number of OSDs::
```